sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestCases/DontMixIncrementOrDecrementWithOtherOperators.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+
+namespace Tests.Diagnostics
+{
+    public class Program
+    {
+        public void Foo()
+        {
+            int val1 = 0;
+            int val2 = 0;
+
+            var result = ++val1 + val2; // Noncompliant {{Extract this [increment|decrement] operation into a dedicated statement.}}
+//                              ^
```

Comment:
Might be better to put the 	squigglie under the ++val1 since that is what the rule is complaining about (also, the squigglie won't be easy to see under a single character).

Reply:
Also, it would be clear if for expressions like ++val1 * ++val2